include/utility/json_utility.hpp

```diff
@@ -606,5 +606,25 @@ inline std::string
     }
     return std::string();
 }
+
+/**
+ * @brief An API to check backup and restore VPD is required.
```

Comment:
need to add details as to the check will check if there is provision for backup and restore mentioned in the json and also if the path is a hardware path by checking the path and size of contents in the path